### DISCUSSION TITLE
How to become a collaborator for llama.cpp?

### DISCUSSION THREAD

**Discussion by VJHack** [ggml-org/llama.cpp#9395] [Q&A] [answered]
Although I am relatively new, I have been keeping up with the latest on faster inference of LLMs on CPUs. I very much believe in the mission of this project, and I'm interested in contributing to drive this project forward.

How can I become a contributor? I'd like to start by resolving smaller issues. To achieve this, I'd assume I would need to be granted collaborator access to this project.

**Comment by VJHack** [comment, marked as answer]
Since I'm very new to llama.cpp and the open source community as a whole, I'm answering this question for any newcomers who might have the same question. Essentially you want to fork this repo, make your changes, and open a open a PR as mentioned [here](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project).

As always, be sure to read the[ contribution guidelines](https://github.com/ggerganov/llama.cpp/blob/master/CONTRIBUTING.md).

**Comment by onur-goksu** [comment]
Hey, I was looking for an open-source repo to contribute and gain experience in new, cutting-edge domains and improve my coding skills while researching and having code-reviews.

I have no experience (nor knowledge) in LLM, although I'm a C/C++ developer with 4.5 years of experience. Do you think this is a right repo to begin with?
